build/bk-build.js

```diff
@@ -205,6 +205,13 @@
         }
       }
 
+      // Copy proxy.conf.js so the front-end is all ready to go against a local backend - if not already exsiting
+      var proxyConf = path.resolve(__dirname, '../proxy.conf.js');
+      var localProxyConf = path.resolve(__dirname, './proxy.conf.localdev.js');
+      if (!fs.existsSync(proxyConf)) {
+        fs.copySync(localProxyConf, proxyConf);
+      }
+
```

Comment:
If there is already a proxy.conf.js we could make a backup by renaming it to `proxy.conf.js.bk` and write out the new one. This would preserve the file in case someone wants to go back to it.

Reply:
This won't overwrite one if there is already one there - this is just there for the first time, of-out-box case.